functions/package.json

```diff
@@ -20,14 +20,15 @@
   },
   "main": "lib/index.js",
   "dependencies": {
-    "firebase": "^6.4.2",
-    "googleapis": "^39.2.0",
-    "nodemailer": "^6.3.0",
     "@google-cloud/storage": "^3.1.0",
     "busboy": "^0.3.1",
     "cors": "^2.8.5",
+    "firebase": "^6.4.2",
     "firebase-admin": "^8.0.0",
     "firebase-functions": "^3.1.0",
+    "gender-api.com-client": "^1.0.14",
```

Comment:
Do you actually use this somewhere? 🤔

Reply:
Actually I also plan to use some gender API to identify the gender of the person based on the name. I was told we also need to identify gender for statistical purposes.